src/Tests/HealthCheck/SessionHealthCheckChainTest.php

```diff
@@ -73,7 +74,7 @@ public function testChainStopsWhenCheckFails()
             ->andReturn(false);
 
         $statusDown = m::mock(HealthReport::buildStatusDown('Lorem ipsum dolor sit'));
-        $statusOk
```

Comment:
@MKodde can you confirm this was an mistake?

The problem is, in both cases the test passes